### PR TITLE
Week3step4

### PR DESCRIPTION
**Pull request by jackdavidweber**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (g-harel) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:55]: Nice!

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (g-harel) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: If the form was empty should an empty comment be added or should it be ignored?

File: portfolio/src/main/webapp/index.html
Comment (g-harel) [inline portfolio/src/main/webapp/index.html:538]: I'm not seeing the matching `<div>`

File: portfolio/src/main/webapp/script.js
Comment (g-harel) [inline portfolio/src/main/webapp/script.js]: ❗

File: portfolio/src/main/webapp/script.js
Comment (g-harel) [inline portfolio/src/main/webapp/script.js]: Can this be deleted now?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (augerm) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: nit/optional: Better to be specific here with something like `commentsJson` or something.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (augerm) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: nit/optional: Param names are usually snake_case or camelCase. 

text_input or textInput

Comment (augerm) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Better yet - user_comment or userComment might be better here.

File: portfolio/src/main/webapp/script.js
Comment (augerm) [inline portfolio/src/main/webapp/script.js]: var --> let

File: portfolio/src/main/webapp/script.js
Comment (augerm) [inline portfolio/src/main/webapp/script.js]: Is this function used to get the comments? If so - I'd recommend sticking to one name rather than introducing `quote`.

Comment (augerm) [reply portfolio/src/main/webapp/script.js]: Also `fetchTutorial` might be best named `commentsSection`

**Review by augerm — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/script.js
Comment (jackdavidweber) [reply portfolio/src/main/webapp/script.js]: yes- In terms of workflow, I already fixed these issues in week3step3 but since i started working on this before merging week3step3, it did not get recognized. What should I do in these cases?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (jackdavidweber) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: I fixed this in step 6

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (jackdavidweber) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: got it

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (jackdavidweber) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: got it

File: portfolio/src/main/webapp/script.js
Comment (jackdavidweber) [reply portfolio/src/main/webapp/script.js]: this is old and i just hadn't deleted yet. I just removed it all

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (jackdavidweber) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: @augerm fixed comments- can you unblock my merge?

File: portfolio/src/main/webapp/script.js
Comment (g-harel) [reply portfolio/src/main/webapp/script.js]: I would suggest you do the same thing, but then only open PR once the preceding one is merged into master.

**Review by augerm — APPROVED** [review]

**Pull request merged**
